### PR TITLE
UPSTREAM: <carry>: adding initial manifest generation

### PR DESCRIPTION
**Pull request by grokspawn**
# Description

## Reviewer Checklist

- [ ] API Go Documentation
- [ ] Tests: Unit Tests (and E2E Tests, if appropriate)
- [ ] Comprehensive Commit Messages
- [ ] Links to related GitHub Issue(s)

### CODE REVIEW

File: openshift/generate-manifests.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+TMP_ROOT="$(mktemp -p . -d 2>/dev/null || mktemp -d ./tmpdir.XXXXXXX)"
+# Make sure to delete the temp dir when we exit
+trap 'rm -rf $TMP_ROOT' EXIT
+
+# Copy all kustomize files into a temp dir
+TMP_CONFIG="${TMP_ROOT}/config"
+cp -a "${REPO_ROOT}/config" "$TMP_CONFIG"
+
+# Override namespace to openshift-operator-controller-system
+$YQ -i '.namespace = "openshift-operator-controller-system"' "${TMP_CONFIG}/default/kustomization.yaml"
```
Comment (joelanford) [inline openshift/generate-manifests.sh:23]: We named the catalogd namespace `openshift-catalogd`, so follow that pattern here?
```suggestion
$YQ -i '.namespace = "openshift-operator-controller"' "${TMP_CONFIG}/default/kustomization.yaml"
```

**Comment by ncdc** [comment]
Manually merging while we work out DOWNSTREAM_OWNERS

**Pull request merged**